Userland/readlink.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2018-2020, sppmacd <sppmacd@pm.me>
```

Comment:
You haven't been working on this in 2018 🙂

Also, use the real name here, perhaps?